neqo-transport/src/connection.rs

```diff
@@ -911,6 +972,19 @@ impl Connection {
                 epoch,
             );
             self.stats.packets_tx += 1;
+
+            // Only reset idle timeout if we've received a packet since the last
+            // time we reset the timeout here.
+            match self.idle_timeout {
```

Comment:
I think that you want a new function for this.  Maybe on `IdleTimeout`.

Reply:
great idea!